<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtPrintRatingDialog</class>
 <widget class="QDialog" name="QtPrintRatingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preview van PDF</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Brainweaver.qrc">
    <normaloff>:/images/PicArrow14x14.png</normaloff>:/images/PicArrow14x14.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>703</width>
        <height>329</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>703</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QFrame" name="frame_header">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_title">
            <property name="text">
             <string>ANALYSE VAN DE CONCEPT MAP VAN DE PERSOONLIJKE WERKTHEORIE</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_focal_question">
            <property name="text">
             <string>FOCUS: ?</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_student_name">
            <property name="text">
             <string> VAN: ?</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_assessor_name">
            <property name="text">
             <string>ASSESSOR: ?</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_date">
            <property name="text">
             <string>Datum: ?</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_concept_map">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_concept_map">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Concept map</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="button_print">
     <property name="text">
      <string>&amp;Sla PDF op</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Brainweaver.qrc"/>
 </resources>
 <connections/>
</ui>
